### PR TITLE
Minor content changes for 2023

### PR DESCRIPTION
**Pull request by benmartin-coforma**
### Description
1. Add calculation explanation to Section 2, Part 1: "Number of Children enrolled in CHIP"
2. Remove past years from Section 2, Part 2. It only needs to go back 5 years, so for 2023, going back to 2018 is sufficient.
3. Add "Part 1" and "Part 2" headers to Section 3I. This is tricky and maybe not the best solution, but Part.js line 55 prevents these headers from displaying unless the part has a `title`. I don't have any text to add for the title but I can add an invisible truthy string: U+00A0, the non-breaking space. This causes the headers to render as `Part 1:` and `Part 2:`. If the trailing colon is not desirable, then this request will require a broader change.
4. Change repeatable question text in Section 3I to "Does the state have additional HSIs to report?". This required a schema change to optionally pass in this text. I don't believe it will affect any other questions.
5. Change repeatable button text in Section 3I to "Add Another HSI Program". The requested text was "Add Another HSI", but the JSON already had `"typeLabel": "HSI Program"`, and an apparent bug: it seems like it was always intended for this text to pass through to the button text. Additionally, it seems like it was always intended to pass this through to the "do you have another" question text, as well as the "delete last" button text. I've "fixed" it here, but ***THIS MAY NOT BE A GOOD IDEA***, as it will affect all repeatables throughout the report. This includes all six repeatables in Section 4, which will now ask "Do you have another Goal in this list?" with button text "Add another Goal".
6. Hide the "Optional" text under "Does the state have additional HSIs to report?". This was another schema change, to optionally pass a `hideOptionalHint` flag through the question.

### Related ticket(s)
MDCT-3035

---
### How to test
1. Log in to CARTS as an admin
2. Generate the 2023 forms
3. Log in to CARTS as a state user
4. Click through the report to observe the changes

### Important updates
n/a

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary

### CODE REVIEW

**Comment by BearHanded** [comment]
> Add "Part 1" and "Part 2" headers to Section 3I. ... If the trailing colon is not desirable, then this request will require a broader change.
This one I could go either way on, if only due to timing. My guess is the trailing colon will be fine. It's worth running by David, though.

File: services/database/data/seed/seed-section-base-2023.json [hunk 1]
```diff
@@ -8661,12 +8517,15 @@
...
                 "id": "2023-03-i-02",
+                "title": "\u00A0",
                 "text": "Please answer the following questions for all of the state's approved HSIs as listed in section 2.2 of the CHIP state plan.",
                 "type": "part",
                 "questions": [
                   {
                     "id": "2023-03-i-02-01",
                     "type": "repeatables",
+                    "addAnotherText": "Does the state have additional HSIs to report?",
+                    "hideOptionalHint": false,
```
Comment (gmrabian) [inline services/database/data/seed/seed-section-base-2023.json:8528]: I think we want this to be true, right?

Screenshot 2023-10-06 at 9 29 22 AM

**Review by gmrabian — CHANGES_REQUESTED** [review]
We want to hide the 'optional' hint, right?
Otherwise changes look good. I verified the trailing colon and non-breaking space don't have any issues when read with VoiceOver (It just says "Part 1" and moves on)

File: services/ui-src/src/components/fields/Repeatables.js [hunk 2]
```diff
@@ -39,6 +39,15 @@ const Repeatables = ({
     removeRepeatableFrom(question.id);
   };
 
+  let doYouHaveAnotherVerbiage;
+  if (question.addAnotherText) {
+    doYouHaveAnotherVerbiage = question.addAnotherText;
+  } else if (question.typeLabel) {
+    doYouHaveAnotherVerbiage = `Do you have another ${question.typeLabel} in this list?`;
+  } else {
+    doYouHaveAnotherVerbiage = `Do you have another in this list?`;
```
[the remark this answers is not in the document]
Comment (benmartin-coforma) [reply services/ui-src/src/components/fields/Repeatables.js:48]: The existing behavior for this particular text omitted any fallback to `type`, so I'm comfortable doing the same.

**Review by gmrabian — APPROVED** [review]

**Pull request merged**